requirements.txt

```diff
@@ -0,0 +1,24 @@
+astropy
+hypothesis==6.29.*
+matplotlib
+more-itertools
+numpy>1.21.*
+openpyxl
+pytest
+pytz
+tabulate
+scipy
+coverage
+requests
+pandas
```

Comment:
You can remove `pandas` after https://github.com/gemini-hlsw/scheduler/pull/165 is merged.